folly/io/async/AsyncSocket.cpp

```diff
@@ -662,7 +662,11 @@ void AsyncSocket::writeChain(WriteCallback* callback, unique_ptr<IOBuf>&& buf,
   constexpr size_t kSmallSizeMax = 64;
   size_t count = buf->countChainElements();
   if (count <= kSmallSizeMax) {
+    // suppress "warning: variable length array ‘vec’ is used [-Wvla]"
+#pragma GCC diagnostic push
```

Comment:
We have those, but we actually only use it in a couple of places. I added them but never got around to switching everything over to them.